visualc/VS2010/aescrypt2.vcxproj

```diff
@@ -95,7 +95,9 @@
       <WarningLevel>Level3</WarningLevel>
       <Optimization>Disabled</Optimization>
       <PreprocessorDefinitions>WIN32;_DEBUG;_CONSOLE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
-      <AdditionalIncludeDirectories>../../include</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>
+../../crypto/include;../../include
+      </AdditionalIncludeDirectories>

```

Comment:
Do these lines have Windows line endings?

Reply:
I checked myself and they appear to have Windows line endings, which is good for these files.

Reply:
I believe so, yes.